quipucords/api/insights_report/tests_insights_report.py

```diff
@@ -0,0 +1,299 @@
+#
+# Copyright (c) 2017-2019 Red Hat, Inc.
```

Comment:
This is a new file right shouldn't it be 2019?

Reply:
😬